QuizApp/QuizApp/DataSources/DataSourceModels.swift

```diff
@@ -1,3 +1,38 @@
+import RealmSwift
+
+class QuizDatabaseModel: Object {
+
+    @Persisted var id: Int
+    @Persisted var name: String
+    @Persisted var desc: String
+    @Persisted var category: String
+    @Persisted var difficulty: String
+    @Persisted var imageUrl: String
+    @Persisted var numberOfQuestions: Int
+
+    convenience init(_ model: QuizRepoModel) {
```

Comment:
Im not sure where to move this since QuizDatabaseModel shouldn't be aware of QuizRepoModel

Reply:
Nvm